tests/cql/CqlIntervalOperatorsTest.xml

```diff
@@ -1,1736 +1,133 @@
-		</test>
-		<test name="IntegerIntervalTest5" version="1.0">
-			<expression>Interval[4, 15]</expression>
-			<output>Interval[4, 15]</output>
-		</test>
-		<test name="DecimalIntervalTest" version="1.0">
-			<expression>Interval[1.0, 10.0]</expression>
-			<output>Interval[1.0, 10.0]</output>
-		</test>
-		<test name="DecimalIntervalTest2" version="1.0">
-			<expression>Interval[11.0, 20.0]</expression>
-			<output>Interval[11.0, 20.0]</output>
-		</test>
-		<test name="DecimalIntervalTest3" version="1.0">
-			<expression>Interval[4.0, 10.0]</expression>
-			<output>Interval[4.0, 10.0]</output>
-		</test>
-		<test name="DecimalIntervalTest4" version="1.0">
-			<expression>Interval[4.0, 15.0]</expression>
-			<output>Interval[4.0, 15.0]</output>
-		</test>
-		<test name="DecimalIntervalTest5" version="1.0">
-			<expression>Interval[14.0, 15.0]</expression>
-			<output>Interval[14.0, 15.0]</output>
-		</test>
-		<test name="QuantityIntervalTest" version="1.0">
-			<expression>Interval[1.0 'g', 10.0 'g']</expression>
-			<output>Interval[1.0 'g', 10.0 'g']</output>
-		</test>
-		<test name="QuantityIntervalTest2" version="1.0">
-			<expression>Interval[11.0 'g', 20.0 'g']</expression>
-			<output>Interval[11.0 'g', 20.0 'g']</output>
-		</test>
-		<test name="QuantityIntervalTest3" version="1.0">
-			<expression>Interval[5.0 'g', 10.0 'g']</expression>
-			<output>Interval[5.0 'g', 10.0 'g']</output>
-		</test>
-		<test name="QuantityIntervalTest4" version="1.0">
-			<expression>Interval[5.0 'g', 15.0 'g']</expression>
-			<output>Interval[5.0 'g', 15.0 'g']</output>
-		</test>
-		<test name="QuantityIntervalTest5" version="1.0">
-			<expression>Interval[14.0 'g', 15.0 'g']</expression>
-			<output>Interval[14.0 'g', 15.0 'g']</output>
-		</test>
-		<test name="DateTimeIntervalTest" version="1.0">
-			<expression>Interval[@2016-05-01T00:00:00.000, @2016-05-02T00:00:00.000]</expression>
-			<output>Interval[@2016-05-01T00:00:00.000, @2016-05-02T00:00:00.000]</output>
-		</test>
-		<test name="TimeIntervalTest" version="1.0">
-			<expression>Interval[@T00:00:00.000, @T23:59:59.599]</expression>
-			<output>Interval[@T00:00:00.000, @T23:59:59.599]</output>
-		</test>
-		<test name="CollapseIntervalTestInteger" version="1.0">
-			<expression>{Interval[1, 10], Interval[11, 20], Interval[44, 50]}</expression>
-			<output>{Interval[1, 10], Interval[11, 20], Interval[44, 50]}</output>
-		</test>
-		<test name="InvalidIntegerInterval" version="1.0">
-			<expression invalid="true">Interval[5, 3]</expression>
-			<!-- EXPECT: Invalid Interval - the ending boundary must be greater than or equal to the starting boundary. -->
-		</test>
-		<test name="InvalidIntegerIntervalA" version="1.0">
-			<expression invalid="true">Interval[5, 5)</expression>
-			<!-- EXPECT: Invalid Interval - the ending boundary must be greater than or equal to the starting boundary. -->
-		</test>
-	</group>
+    <test name="IntegerIntervalAfterFalse" version="1.0">
```

Comment:
This one not only has formatting changes, but it looks like a whole bunch of tests are just gone?